File: airline_code_lookup/v1/get/Python SDK/airline_code_lookup.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    What's the airline name for the IATA code BA?
    '''
    response = amadeus.reference_data.airlines.get(airlineCodes='BA')
    print(response.data)
except ResponseError as error:
    raise error

File: airline_code_lookup/v1/get/Node SDK/airline_code_lookup.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What's the airline name for the IATA code BA?
    const response = await amadeus.referenceData.airlines.get({
      airlineCodes: "BA",
    });
    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
